server/dockerCompose/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.9'
+
+x-app: &default-app
```

Comment:
if this only applies restart: always, I think the default-app thing is more confusing than helpful